uitools/import/Esri/ArcGISRuntime/Toolkit/TimeSlider.qml

```diff
@@ -245,6 +262,9 @@ Pane {
             Layout.alignment: Qt.AlignRight  | Qt.AlignBottom
             Layout.fillWidth: true
             Layout.margins: 5
+            wrapMode: Text.WordWrap
+            clip: true
+            maximumLineCount: 3
```

Comment:
Could maybe add `elide: Text.ElideLeft?`
 https://doc.qt.io/qt-5/qml-qtquick-text.html#elide-prop

Reply:
I tried using it earlier, but only the last string will be elided. And Only in case of ElideRight, so in the demo I m using both ElideRight

https://user-images.githubusercontent.com/91129623/161112398-842e8a09-da2c-41ab-a7a0-4d77d8976ba3.mp4

.